src/components/custom/_dtrading.tsx

```diff
@@ -183,13 +181,17 @@ const DTrading = ({ trading, reverse, two_title }: DTradingProps) => {
                                 <StyledHeader type="page-title" as="h2">
                                     {item.title}
                                 </StyledHeader>
-                                <StyledTitle>{item.subtitle}</StyledTitle>
+                                <StyledTitle weight="400" as="p">
```

Comment:
please provide `type='paragraph-1'` here and instead of `weight="400"` let's move it in styled components as `font-weight: normal`